packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -1017,8 +1019,30 @@ class _InkResponseState extends State<_InkResponseStateWidget>
     }
   }
 
+  void _setInkFeature(bool active) {
+    final List<InkFeature?> inkFeatures = <InkFeature?>[];
+    if (_splashes != null)
+      inkFeatures.addAll(_splashes!);
+    inkFeatures.addAll(_highlights.values);
+
+    for (final InkFeature? splash in inkFeatures)
+      splash?.active = active;
```

Comment:
You could also write this as something like:
```dart
   for (InkFeature splash in [ ...?splashes, ..._hightlights.values ])
     splash.active = active;
```
Not sure which is better, up to you.

Reply:
`...?_splashes`
This is awesome ！！！！！I didn't know this was possible before.